package.json

```diff
@@ -13,12 +13,19 @@
     "start": "next start"
   },
   "dependencies": {
+    "@react-pdf/styled-components": "^1.4.0",
```

Comment:
Is this package required? styled-components should be the only required one AFAIK